client/messaging/message-records.ts

```diff
@@ -1,75 +1,37 @@
 import { Record } from 'immutable'
+import { ChatMessage } from '../chat/chat-message-records'
+import { LobbyMessage } from '../lobbies/lobby-message-records'
 
-export enum ChatMessageType {
+/**
+ * NOTE(2Pac): A common message type that's used in all messaging-related services (e.g. chat,
+ * whispers, lobbies, parties, etc.). For now that's only a default text message, but there might be
```

Comment:
I'd leave out this 'For now...' sentence, it's not particularly useful compared to the source itself and is likely to be left around long past being true